Task2/Project/DataControl/ViewModel/ApplicationViewModel.cs

```diff
@@ -189,49 +211,174 @@ public System.Collections.Generic.IEnumerable<string> RedoActionNames
         public RelayCommand ChangeStrokeWidth => changeStrokeWidth;
 
         public RelayCommand ChangeShapeLocation => changeShapeLocation;
+        #endregion
 
-        
 
         // METHODS
         private void NewFileMethod(object o)
         {
-            throw new System.NotImplementedException();
+            canvas.Clear();
+            OnPropertyChanged(new PropertyChangedEventArgs("dataChanged"));
+        }
+        private void OpenFileMethod(object o)
+        {
+            try
+            {
+                if (dialogService.OpenFileDialog()) 
+                {
+                    currentFileName = dialogService.FilePath;
+                    fileService.Load(canvas, currentFileName);
+                    dialogService.ShowMessage("File is opened and data is loaded from it.");

```

Comment:
show message after action has been done is bad idea, too many messages would be shown.

Reply:
also do not forget OnPropertyChanged

Reply:
Are all such messages unnecessary?

Reply:
yep, they only are necessary if exception happens